source/standards/threat-modelling.html.md.erb

```diff
@@ -0,0 +1,151 @@
+---
+title: Threat Modelling
+last_reviewed_on: 2020-09-28
+review_in: 6 months
+---
+
+# <%= current_page.data.title %>
+
+## What's a threat?
+
+To complete threat modelling, you’ll need to know what assets you are trying to protect. It could be a user database, a deployment pipeline or the integrity of web form submissions.
```

Comment:
I think we should say something about threat modelling as a continuous and ongoing activity, rather than something which can be "completed". If a software system evolves and changes over time the threat model will change. Even if the system doesn't change, the external threats change and evolve. So, I think it would be good to say something about building in a culture of threat modelling into how the team thinks about their day-to-day work.

Reply:
That's a good point, thanks David. I will change that wording and also related, add more info on the 'when'.

Reply:
Hey Ollie, have you managed to update the wording here? Couldn't see anything about continuous and ongoing activity as David suggests? Cheers, Stephen